### ISSUE TITLE
Ctrl+Shift+Up/Down should do text selection, not go up/down the notes list

### ISSUE THREAD

**Issue by uncertainquark** [Automattic/simplenote-electron#798]
#### Steps to reproduce
1. Open a note
2. Select text using Ctrl+Shift+Left/Right keys, it works
3. Now try selecting text using Ctrl+Shift+Up/Down keys, doesn't work

#### What I expected
The text selection would select the entire line as in how all office programs, text editors and note apps function.

#### What happened instead
The notes list got navigated instead using the said key combination which is not in line with any office program, text editor or other notes apps.

#### OS version
Ubuntu 18.04

**Comment by triangulum** [comment]
Could we maybe add <kbd>CTRL</kbd>+<kbd>PgUp</kbd> to go up, and <kbd>CTRL</kbd>+<kbd>PgDn</kbd>? That would be in line with Gnome user conventions (such as scrolling through tabs in Nautilus/Terminal/Chrome/etc).

**Comment by uncertainquark** [comment]
Yes, that sounds good. It will also free up `Ctrl + `Shift + `Up/`Down for text selection, which is the default behaviour in every OS.

**Comment by belcherj** [comment]
Thanks!  I’ve added this to our feature requests!

**Comment by codebykat** [comment]
This was updated and now selects text. We use J and K keys for note list navigation. #1983
